Utils/RecomputeCollectionAreas.py

```diff
@@ -0,0 +1,118 @@
+""" Go through all directories and recompute the .json collection areas files."""
+
+
+
+import os
+import multiprocessing
+
+from RMS.Routines.MaskImage import getMaskFile
+from RMS.ConfigReader import loadConfigFromDirectory
+from RMS.Formats import Platepar
+from Utils.Flux import getCollectingArea, FluxConfig
+
+
+def updateCollectionAreaNight(dir_path, flux_config):
+
+    try:
+        # Load the config file
+        config = loadConfigFromDirectory(['.'], dir_path)
+
+        # Load the platepar
+        platepar = Platepar.Platepar()
+        platepar.read(os.path.join(dir_path, config.platepar_name), use_flat=config.use_flat)
+
+        # Load the mask
+        mask = getMaskFile(dir_path, config)
+
```

Comment:
Are we not repeating the initial error here? Looking for platepar and mask in dir_path? Or is it meant to be run on a different folder structure?

Reply:
When running live, the code will already pass the correct mask and the platepar. When post-processing, RMS has to use the platepar and mask in the directory. E.g. imagine I'm reprocessing data from 2 years ago when the mask and platepar were different.

Reply:
I see the potential confusion - the directories this is made to reprocess are ArchivedFiles directories, not CapturedFiles.

Reply:
Oh, I see. Running this code on the Archived folder is fine. As long as the folder contains the appropriate platepar and mask without fail the code is fine.